src/main/java/racing/web/controller/RacingController.java

```diff
@@ -26,34 +29,45 @@ public RacingController(RacingGameService racingGameService) {
     }
 
     @PostMapping("/plays")
-    public ResponseEntity<RacingGameResultResponse> playGame(@RequestBody RacingGameInfoRequest request) {
+    public ResponseEntity<RacingGameSaveResponse> playGame(@RequestBody RacingGameInfoRequest request) {
         RequestValidator<RacingGameInfoRequest> requestValidator = new RacingGameInfoRequestValidator();
         requestValidator.validate(request);
 
-        Cars cars = CarFactory.carFactory(request.getNames());
-        Long gameId = racingGameService.saveGameByCount(request.getCount());
-        racingGameService.playGame(gameId, cars);
+        Long gameId = racingGameService.playNewGame(request.getCount(), request.getNames());
 
-        return ResponseEntity.ok(getRacingGameResultResponse(cars));
+        return ResponseEntity
+                .status(HttpStatus.CREATED)
+                .body(new RacingGameSaveResponse(gameId));
```

Comment:
https://github.com/woowacourse/jwp-racingcar/pull/129#discussion_r1174354620 에서도 남겼지만 더 원칙적으로 구성한다면 다음과 같이 처리할 수도 있습니다.

```suggestion
        return ResponseEntity
                .status(HttpStatus.CREATED)
                .location(URI.create("자원의 식별자 혹은 위치"))
                .build();
```